src/components/inline/Button/Button.module.css

```diff
@@ -1,9 +1,9 @@
 .button {
-  background-color: var(--color-brand);
-  border: 1px solid var(--color-brand);
+  background-color: var(--color-sheltertech-blue);
+  border: 1px solid var(--color-sheltertech-blue);
   color: var(--color-white);
   cursor: pointer;
-  font-family: var(--font-family-base);
+  font-family: var(--font-family-body);
```

Comment:
Should change to  font-family: var(--font-action); this commit should be merged before https://github.com/ShelterTechSF/sheltertech.org/pull/61 so that commit can rebase to these changes.

Reply:
I talked to @richardxia , these changes are so small that changes in the Button component can be here.